src/utils/networkUtils.ts

```diff
@@ -147,4 +147,11 @@ export function onMetamaskNetworkChange(cb: () => void) {
 	window.ethereum.on('networkChanged', listener);
 }
 
+export function hasMetamaskInstalled() {
+	if (typeof window.ethereum !== 'undefined') {
+		return true;
+	}
+	return false;
```

Comment:
I believe this check is for "Web3 Browser Detection", but if you want to be more specific, you should check for "isMetaMask" - https://docs.metamask.io/guide/getting-started.html#detecting-metamask.